libbeat/processors/add_cloud_metadata/add_cloud_metadata.go

```diff
@@ -31,10 +31,17 @@ const (
 
 	// Google GCE Metadata Service
 	gceMetadataURI = "/computeMetadata/v1/?recursive=true&alt=json"
+
+	// Tencent Clound Metadata Service
+	qcloudMetadataHost          = "metadata.tencentyun.com"
+	qcloudMetadataInstanceIdURI = "/meta-data/instance-id"
+	qcloudMetadataRegionURI     = "/meta-data/placement/region"
+	qcloudMetadataZoneURI       = "/meta-data/placement/zone"
```

Comment:
AFAIK, there is no plan for Tentcent Qcloud endpoints to support `/v1/xxx`,  the metadata service is maintained in my team :)
And the other provider AliCloud also has no versioned endpoints neither, at least no indications from the Chinese documents.